### PR TITLE
Rm 128

### PR DESCRIPTION
**Pull request by Alinadimmokdad**
gl-segments

### CODE REVIEW

File: src/pages/gl-segments/index.js [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+  const del = async obj => {
+    try {
+      await postRequest({
+        extension: GeneralLedgerRepository.Segments.del,
+        record: JSON.stringify(obj)
+      })
+      invalidate()
+      toast.success(platformLabels.Deleted)
+    } catch (error) {}
```
Comment (RebeccaFrem02) [inline src/pages/gl-segments/index.js:89]: no need for try catch

File: src/pages/gl-segments/index.js [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+        <GridToolbar
+          onAdd={add}
+          maxAccess={access}
+          labels={_labels}
+          middleSection={
+            <Grid item sx={{ display: 'flex', mr: 2 }}>
+              <CustomComboBox
+                endpointId={SystemRepository.Defaults.qry}
+                parameters={`_filter=GLACSeg`}
+                sx={{ width: 450 }}
```
Comment (RebeccaFrem02) [inline src/pages/gl-segments/index.js:162]: i the styling here needed?

File: src/pages/gl-segments/form/SegmentForm.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+export default function SegmentForm({ labels, obj, maxAccess, formikSegmentId, fetchGridData }) {
+  const { platformLabels } = useContext(ControlContext)
+
+  const { getRequest, postRequest } = useContext(RequestsContext)
+
+  const invalidate = useInvalidate({
+    endpointId: GeneralLedgerRepository.Segments.qry
+  })
+
+  console.log(formikSegmentId, 'ssss')
```
Comment (RebeccaFrem02) [inline src/pages/gl-segments/form/SegmentForm.js:25]: remove console

File: src/pages/gl-segments/index.js (hunk 2 above)
Comment (RebeccaFrem02) [reply src/pages/gl-segments/index.js:162]: image
both ways we need it smaller

Comment (Alinadimmokdad) [reply src/pages/gl-segments/index.js:162]: yes 
image

**Review by RebeccaFrem02 — APPROVED** [review]

**Pull request merged**